app/assets/javascripts/test/snapshots/public/test-bundle/test/libs/nml.spec.js.md

```diff
@@ -1,126 +0,0 @@
-# Snapshot report for `public/test-bundle/test/libs/nml.spec.js`
```

Comment:
You accidentally deleted the NML snapshots, please restore them :)
Good thing is, this won't happen easily again in the future, thanks to @fm3s recent PR (https://github.com/scalableminds/webknossos/pull/3052) :tada: